tap_wrike/client.py

```diff
@@ -86,7 +84,7 @@ def get_access_token(self):
             "client_id": self.client_id,
             "client_secret": self.client_secret,
             "refresh_token": self.refresh_token,
-            "scope": "wsReadOnly,dataExportFull"
+            "scope": "wsReadOnly,dataExportFull,wsReadWrite,amReadOnlyGroup"
```

Comment:
⚠️ I don't like this. Why does the tap need write access? It's a security risk in my opinion @SBurwash

Reply:
Done